domains/transfer/Transfer.API/API/Transfer/Api/Services/ProjectOriginWalletService.cs

```diff
@@ -46,7 +46,7 @@ private async Task<WalletDepositEndpoint> GetWalletDepositEndpoint(string bearer
         var request = new CreateWalletDepositEndpointRequest();
         var headers = new Metadata
         {
-            { "Authorization", bearerToken }
+            { "Authorization", bearerToken.Contains("Bearer ") ? bearerToken : "Bearer " + bearerToken }
```

Comment:
The main problem is that the code base is using `string` as a type for bearer token, which leads to mistakes. 
I would recommend a more type safe solution than this or that your service class knows how to get the bearer token (either from the http request or by creating the bearer token depending on the use case).
An example of a type safe solution could be https://learn.microsoft.com/en-us/dotnet/api/system.net.http.headers.authenticationheadervalue?view=net-8.0 - but there are others ways.